uniffi_bindgen/src/bindings/python/gen_python/mod.rs

```diff
@@ -298,10 +299,13 @@ pub struct PythonWrapper<'a> {
     type_imports: BTreeSet<ImportRequirement>,
 }
 impl<'a> PythonWrapper<'a> {
-    pub fn new(config: Config, ci: &'a ComponentInterface) -> Self {
+    pub fn new(config: Config, ci: &'a mut ComponentInterface) -> Self {
         let type_renderer = TypeRenderer::new(&config, ci);
         let type_helper_code = type_renderer.render().unwrap();
         let type_imports = type_renderer.imports.into_inner();
+
+        ci.apply_naming_conventions(PythonCodeOracle::default());
```

Comment:
Thanks for having a poke at this! Another approach (which may or may not work :) is some technique for "walking/visiting" a CI - possibly even creating a new CI as we go rather than mutating a single one. That might allow us to avoid having the interface itself be aware of any new traits etc. If you squint it ends up looking a lot like our ["external type converter"](https://github.com/mozilla/uniffi-rs/blob/main/uniffi_meta/src/group.rs#L88)